git_repo/services/service.py

```diff
@@ -156,6 +156,7 @@ def __init__(self, r=None, c=None):
                                                       c.get('privatekey', None))))
         self._alias = c.get('alias', self.name)
         self.fqdn = c.get('fqdn', self.fqdn)
+        self.insecure = c.get('insecure', False)
```

Comment:
well that's easy to implement:

``` python
self.insecure = c.get('insecure', '').lower() in ('true', 'yes', '1')
```

then every other value will be considered as false.